src/scripts/getConnectathonBundles.js

```diff
@@ -58,7 +58,6 @@ async function main() {
           if (
             res.resource.resourceType != 'Measure' &&
             res.resource.resourceType != 'Library' &&
-            res.resource.resourceType != 'ValueSet' &&
```

Comment:
Do we want all connectathon bundle valuesets to be added? Just a double check since obviously we decided to filter valuesets out at some point, so I wasn't sure if this would cause any other problems.

Reply:
I think we do, because we need to be able to query mongo to determine if the code is present in the valueSet